core/domain/skill_jobs_one_off_test.py

```diff
@@ -223,9 +223,9 @@ def _mock_get_skill_by_id(unused_skill_id):
         # If the skill had been successfully migrated, this would include a
         # 'successfully migrated' message. Its absence means that the skill
         # could not be processed.
-        expected = [[u'validation_error',
-                     [u'Skill %s failed validation: \'unicode\' object has '
-                      'no attribute \'validate\'' % (self.SKILL_ID)]]]
+        expected = [['validation_error',
```

Comment:
Why was this error changed (for context)?

Reply:
The previous test was quite useless since `_mock_get_skill_by_id` returned string and I have changed it to return proper skill domain object and that is why the error was changed.